UI/AppKit/Interface/LadybirdWebView.mm

```diff
@@ -1072,8 +1072,78 @@ - (void)keyDown:(NSEvent*)event
         return;
     }
 
-    auto key_event = Ladybird::ns_event_to_key_event(Web::KeyEvent::Type::KeyDown, event);
-    m_web_view_bridge->enqueue_input_event(move(key_event));
+    [self interpretKeyEvents:@[ event ]];
+}
+
+- (void)insertText:(id)string replacementRange:(NSRange)replacementRange
```

Comment:
Organizationally, instead of putting these methods in between the `keyDown` and `keyUp` overrides, let's move them to their own section below prefixed with a mark:

```c++
#pragma mark - NSTextInputClient
```

The mark just makes it clearer which methods belong to which interface, and marks show up in code navigation in some IDEs